src/mds/mdstypes.h

```diff
@@ -181,18 +182,24 @@ struct frag_info_t : public scatter_info_t {
   }
 
   // *this += cur - acc;
-  void add_delta(const frag_info_t &cur, frag_info_t &acc, bool& touched_mtime) {
+  void add_delta(const frag_info_t &cur, frag_info_t &acc, bool& touched_mtime, bool& touched_chattr) {
```

Comment:
Since you're touching it, can you switch this to use pointers for the out params? That's what our style guide calls for, although we obviously have a bunch of old "naughty" code like you're seeing here.